_pages/ato.md

```diff
@@ -2,25 +2,16 @@
 title: ATOs
 ---
 
-An Authority to Operate (ATO) is a complex security review process that is required before you can deploy anything on the public web. See also: [the more formal definition](https://github.com/fisma-ready/introduction#introduction).
-
-Before a system is made publicly accessible on the Internet, it must go through either the full ATO process or a 90-Day [Lightweight Authority to Operate (LATO)](types/). Systems can be made available for use in an extremely limited capacity if they meet the [pre-authorization criteria](types/#pre-authorization).
-
-While the ATO is the final compliance step that's required before launching an application, you will want to **start the process as soon as possible** after your project gets off the ground.
+Every federal information system must go through NIST's [Risk Management Process](background/) before it can be used to process federal information. This process culminates in a signed Authority to Operate (ATO) being issued. Because the Risk Management Process is a complex, multi-step process which will constrain the design and implementation of your system, you should start thinking about how it applies to your system _before_ you begin designing and implementing it. The steps of the Risk Management Process should be executed in parallel with other project activities. Please get in touch with the TTS Infrastructure Team as soon as your project gets funding so you can discuss which category of ATO will be required for your system, and understand what will be required to gain an ATO for it.
```

Comment:
This should use "Risk Management Framework" terminology consistently, as NIST RMF is widely referenced, so that people don't think "Risk Management Process" is something else.